apps/sms/views/conversation/js/conversation.js

```diff
@@ -927,13 +927,13 @@ var ConversationView = {
       return;
     }
 
-    MessageManager.markMessagesRead([message.id]);
-
-    this.onMessage(message);
-    this.scrollViewToBottom();
-    if (this.isScrolledManually) {
-      this.showNewMessageNotice(message);
-    }
+    MessageManager.markMessagesRead([message.id]).then(() => {
```

Comment:
I don't think we want to do this. We don't need to wait that the message is mark as read before doing the rest of the work.